common/task_executor/src/rayon_manager.rs

```diff
@@ -0,0 +1,57 @@
+use rayon::{ThreadPool, ThreadPoolBuilder};
+use std::sync::Arc;
+
+const DEFAULT_LOW_PRIORITY_CPU_PERCENTAGE: usize = 25;
+const MINIMUM_LOW_PRIORITY_THREAD_COUNT: usize = 1;
+
+const DEFAULT_HIGH_PRIORITY_CPU_PERCENTAGE: usize = 80;
+const MINIMUM_HIGH_PRIORITY_THREAD_COUNT: usize = 3;
```

Comment:
I think we could keep this as 1 as well. 
Supernodes on live network should always have more than this, however this may not be the case for devnets, smaller network or integration tests